contrib/quickstart/kratos/email-password/kratos.yml

```diff
@@ -65,6 +65,7 @@ selfservice:
         password:
           hooks:
             - hook: session
+            - hook: show_verification_ui
```

Comment:
Maybe continue_with_verification_ui for naming consistency?

Reply:
Also possible, though the key in a client application's code would be `show_verification_ui` with https://github.com/ory/kratos/pull/3144#discussion_r1142064211

Reply:
:D nice timing. Alright. Gonna disregard this.